CHANGELOG.md

```diff
@@ -1,40 +0,0 @@
-# Change Log
```

Comment:
This Change log meant to be removed?

Reply:
Yes, as it was of no value (as it's empty). Might be a bit drastical for a PR.

However, you can decide otherwise and keep the changelog.